_timeline/2018-04-26-mesh-node-software-releases.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Mesh Node Software Releases
+date: 2018-04-26
+---
+Released [Prototype v0.2](https://github.com/tomeshnet/prototype-cjdns-pi/releases/tag/v0.2) with new features and support for many new single-board computers (SBCs). [Mesh Orange](https://github.com/tomeshnet/mesh-orange), a Debian-based system image for SBC-based mesh routers, has matured to become the operating system for our [Peer-to-Peer Internet](https://github.com/tomeshnet/p2p-internet-workshop) workshop series, and [pre-compiled images](https://github.com/benhylau/mesh-router-builder/releases) are now available.
```

Comment:
Is SBC common format? Just think you could cut the acronym the second time to avoid introing it. 

Also, some small tweaks to reduce words? Not sure if you need the full workshop shout-out here

`Released [Prototype v0.2](https://github.com/tomeshnet/prototype-cjdns-pi/releases/tag/v0.2) with new features and support for more single-board computers. [Mesh Orange](https://github.com/tomeshnet/mesh-orange), a Debian-based system image, matured to become the operating system for our [Peer-to-Peer Internet](https://github.com/tomeshnet/p2p-internet-workshop) workshop and has [pre-compiled images](https://github.com/benhylau/mesh-router-builder/releases).`